zcash_client_sqlite/src/wallet/transparent.rs

```diff
@@ -383,16 +383,61 @@ pub(crate) fn get_transparent_balances<P: consensus::Parameters>(
 pub(crate) fn add_transparent_account_balances(
     conn: &rusqlite::Connection,
     mempool_height: BlockHeight,
+    min_confirmations: u32,
     account_balances: &mut HashMap<AccountId, AccountBalance>,
 ) -> Result<(), SqliteClientError> {
```

Comment:
```suggestion
) -> Result<(), SqliteClientError> {
    // TODO (#issue number): distinguish between pending change and non-change.

```
(if this is not intended to be fixed in this PR).